cas/service.c

```diff
@@ -1148,6 +1186,15 @@ static int cas_fom_tick(struct m0_fom *fom0)
 		if (cas_in_ut() && m0_fom_phase(fom0) == M0_FOPH_QUEUE_REPLY) {
 			m0_fom_phase_set(fom0, M0_FOPH_TXN_COMMIT_WAIT);
 		}
+		if (m0_fom_phase(fom0) == M0_FOPH_FINISH) {
```

Comment:
For better understanding of the code block here, I'd like someone to add comments after calling "m0_fom_tick_generic(fom0)", to say that,
```
m0_fom_phase(fom0) is changed, and now it is the next phase.
```

Reply:
Rewritten.